### PR TITLE
feat!: take in latest a4u (v3.5.3) release

### PR DESCRIPTION
**Pull request by pfulton**
## Description

Takes in the latest release from `@a4u/a4u-s2-icon-global-set-open-source-processed` which is `3.5.3`: https://artifactory.corp.adobe.com/ui/packages/npm:%2F%2F@a4u%2Fa4u-s2-icon-global-set-open-source-processed/3.5.3?name=%40a4u%2Fa4u-s2-icon-global-set-open-source-processed&type=packages

It's difficult to understand what's changed and/or what's new, but here's a list of what's in this release along with a "modified date" and a "added to package" date: https://a4u-gateway.corp.adobe.com/bundles/6603e959248c150261362b85/overview

Previously, we had consumed `v2.0.0`, and here are the details from the changelog between `2.0.0` and `3.5.3`:

<details>
<summary>Toggle to show or hide the changelog</summary>

# CHANGELOG
## [3.5.3]
Date: Fri Aug 30 2024 16:46:33 GMT+0200 (Central European Summer Time)

Comment: Little artwork adjustments

### Changed
- S2_Icon_GlobeGrid_20_N.svg
- S2_Icon_Market_20_N.svg
- S2_Icon_Settings_20_N.svg

## [3.5.2]
Date: Fri Aug 23 2024 21:48:02 GMT+0200 (Central European Summer Time)

Comment: more XML corrections

### Changed
- S2_Icon_DataSettings_20_N.svg

## [3.5.1]
Date: Fri Aug 23 2024 21:40:06 GMT+0200 (Central European Summer Time)

Comment: Some XML corrections

### Changed
- S2_Icon_CheckmarkCircle_20_N.svg
- S2_Icon_CheckmarkSize300_20_N.svg
- S2_Icon_DataAdd_20_N.svg
- S2_Icon_DataUpload_20_N.svg

## [3.5.0]
Date: Fri Aug 23 2024 18:51:17 GMT+0200 (Central European Summer Time)

Comment: Added more icons

### Added
- S2_Icon_ChevronLeft_20_N.svg
- S2_Icon_ChevronRight_20_N.svg
- S2_Icon_GradientHorizontal_20_N.svg
- S2_Icon_GradientRadial_20_N.svg
- S2_Icon_Gradient_20_N.svg
- S2_Icon_TextCapsAll_20_N.svg
- S2_Icon_TextCapsSmall_20_N.svg
- S2_Icon_ViewTransparency_20_N.svg

## [3.4.0]
Date: Fri Aug 23 2024 16:23:40 GMT+0200 (Central European Summer Time)

Comment: Adding close icons

### Added
- S2_Icon_CloseCircle_20_N.svg
- S2_Icon_Close_20_N.svg

## [3.3.0]
Date: Thu Aug 22 2024 19:45:59 GMT+0200 (Central European Summer Time)

Comment: Added more icons

### Added
- S2_Icon_Channel_20_N.svg
- S2_Icon_CheckmarkSize300_20_N.svg
- S2_Icon_ChevronDownSize300_20_N.svg
- S2_Icon_CursorClick_20_N.svg
- S2_Icon_DataAdd_20_N.svg
- S2_Icon_DataSettings_20_N.svg
- S2_Icon_DataUpload_20_N.svg
- S2_Icon_Data_20_N.svg
- S2_Icon_EffectBorder_20_N.svg
- S2_Icon_Feedback_20_N.svg
- S2_Icon_NamingOrder_20_N.svg
- S2_Icon_Table_20_N.svg

## [3.2.0]
Date: Wed Aug 21 2024 13:01:24 GMT+0200 (Central European Summer Time)

Comment: Adding more icons and updating some artworks

### Added
- S2_Icon_FolderMoveTo_20_N.svg
- S2_Icon_Ruler_20_N.svg
- S2_Icon_SelectMulti_20_N.svg
- S2_Icon_SelectNone_20_N.svg

### Changed
- S2_Icon_AddCircle_20_N.svg
- S2_Icon_CheckmarkCircle_20_N.svg
- S2_Icon_ColorFill_20_N.svg
- S2_Icon_FolderOpen_20_N.svg
- S2_Icon_LockOpen_20_N.svg
- S2_Icon_Lock_20_N.svg
- S2_Icon_RemoveCircle_20_N.svg
- S2_Icon_Shapes_20_N.svg
- S2_Icon_Target_20_N.svg

## [3.1.0]
Date: Thu Jul 11 2024 16:36:13 GMT+0200 (Central European Summer Time)

Comment: Adding 5 more icons

### Added
- S2_Icon_AlertDiamond_20_N.svg
- S2_Icon_AlignBottom_20_N.svg
- S2_Icon_ShareAndroid_20_N.svg
- S2_Icon_StrokeSolid_20_N.svg
- S2_Icon_TransformDistort_20_N.svg

## [3.0.0]
Date: Mon Jul 08 2024 17:43:59 GMT+0200 (Central European Summer Time)

Comment: Big update, renamed some icons, removed some icons added icons

### Added
- S2_Icon_3DMaterial_20_N.svg
- S2_Icon_AddCircle_20_N.svg
- S2_Icon_BadgeVerified_20_N.svg
- S2_Icon_Blur_20_N.svg
- S2_Icon_Brand_20_N.svg
- S2_Icon_CameraProperties_20_N.svg
- S2_Icon_Cancel_20_N.svg
- S2_Icon_CloseCaptions_20_N.svg
- S2_Icon_Collection_20_N.svg
- S2_Icon_ColorHarmony_20_N.svg
- S2_Icon_CommentShow_20_N.svg
- S2_Icon_ContextualTaskBar_20_N.svg
- S2_Icon_CornerRadiusBottomLeft_20_N.svg
- S2_Icon_CornerRadiusBottomRight_20_N.svg
- S2_Icon_CornerRadiusTopLeft_20_N.svg
- S2_Icon_CornerRadiusTopRight_20_N.svg
- S2_Icon_CropRotate_20_N.svg
- S2_Icon_DeviceMultiscreen_20_N.svg
- S2_Icon_EditNo_20_N.svg
- S2_Icon_Effects_20_N.svg
- S2_Icon_Email_20_N.svg
- S2_Icon_Exposure_20_N.svg
- S2_Icon_FindAndReplace_20_N.svg
- S2_Icon_FontPicker_20_N.svg
- S2_Icon_GridTypeDots_20_N.svg
- S2_Icon_GridTypeLines_20_N.svg
- S2_Icon_GridsAndRulers_20_N.svg
- S2_Icon_GroupNo_20_N.svg
- S2_Icon_History_20_N.svg
- S2_Icon_ImageAdd_20_N.svg
- S2_Icon_Images_20_N.svg
- S2_Icon_Invert_20_N.svg
- S2_Icon_Leave_20_N.svg
- S2_Icon_ListBulleted_20_N.svg
- S2_Icon_ListMultiSelect_20_N.svg
- S2_Icon_Logo_20_N.svg
- S2_Icon_Market_20_N.svg
- S2_Icon_MaskDisable_20_N.svg
- S2_Icon_Mask_20_N.svg
- S2_Icon_MediaOffline_N.svg
- S2_Icon_MusicNote_20_N.svg
- S2_Icon_OrientationLandscape_20_N.svg
- S2_Icon_OrientationPortrait_20_N.svg
- S2_Icon_Path_20_N.svg
- S2_Icon_Pattern_20_N.svg
- S2_Icon_Pause_20_N.svg
- S2_Icon_PenBrush_20_N.svg
- S2_Icon_People_20_N.svg
- S2_Icon_Play_20_N.svg
- S2_Icon_ProjectAddInto_20_N.svg
- S2_Icon_Project_20_N.svg
- S2_Icon_PublishNo_20_N.svg
- S2_Icon_RectangleHoriz_20_N.svg
- S2_Icon_RemoveCircle_20_N.svg
- S2_Icon_Ribbon_20_N.svg
- S2_Icon_RocketQuickActions_20_N.svg
- S2_Icon_Saturation_20_N.svg
- S2_Icon_SelectAllItems_20_N.svg
- S2_Icon_SelectNo_20_N.svg
- S2_Icon_SelectRectangle_20_N.svg
- S2_Icon_Shuffle_20_N.svg
- S2_Icon_Similar_20_N.svg
- S2_Icon_SocialNetwork_20_N.svg
- S2_Icon_StepBackward_20_N.svg
- S2_Icon_StepForward_20_N.svg
- S2_Icon_Target_20_N.svg
- S2_Icon_TextAdd_20_N.svg
- S2_Icon_TextAlignJustifyLastRight_20_N.svg
- S2_Icon_TextNumbers_20_N.svg
- S2_Icon_TextParagraph_20_N.svg
- S2_Icon_TextVariableFontSettings_20_N.svg
- S2_Icon_TouchOneFingerSwipeLeftRight_20_N.svg
- S2_Icon_Transcript_20_N.svg
- S2_Icon_TransformPerspective_20_N.svg
- S2_Icon_TransformSkew_20_N.svg
- S2_Icon_TransformWarp_20_N.svg
- S2_Icon_Translate_20_N.svg
- S2_Icon_Tutorials_20_N.svg
- S2_Icon_Upload_20_N.svg
- S2_Icon_UserEdit_20_N.svg
- S2_Icon_UserFollowing_20_N.svg
- S2_Icon_ViewGrid_20_N.svg
- S2_Icon_ViewList_20_N.svg

### Changed
- S2_Icon_Asset_20_N.svg
- S2_Icon_BetaApp_20_N.svg
- S2_Icon_DeviceTablet_20_N.svg
- S2_Icon_Refresh_20_N.svg
- S2_Icon_Shapes_20_N.svg
- S2_Icon_StarFilled_20_N.svg
- S2_Icon_Star_20_N.svg

### Deleted
- S2_Icon_AddToSelection_20_N.svg
- S2_Icon_AddTo_20_N.svg
- S2_Icon_AlertCircle_20_N.svg
- S2_Icon_AlignBottom_20_N.svg
- S2_Icon_AlignOnGrid_20_N.svg
- S2_Icon_AllCaps_20_N.svg
- S2_Icon_AppGear_20_N.svg
- S2_Icon_AppsInYourPlan_20_N.svg
- S2_Icon_BuildingsGear_20_N.svg
- S2_Icon_ClockEdit_20_N.svg
- S2_Icon_CornerCount_20_N.svg
- S2_Icon_Deselect_20_N.svg
- S2_Icon_Distort_20_N.svg
- S2_Icon_DividePath_20_N.svg
- S2_Icon_ExcludeOverlap_20_N.svg
- S2_Icon_GridView_20_N.svg
- S2_Icon_HealSpot_20_N.svg
- S2_Icon_Heal_20_N.svg
- S2_Icon_Illustration_20_N.svg
- S2_Icon_IntersectOverlap_20_N.svg
- S2_Icon_Length_20_N.svg
- S2_Icon_ListView_20_N.svg
- S2_Icon_MagneticLine_20_N.svg
- S2_Icon_MarginTemplate_20_N.svg
- S2_Icon_NoEdit_20_N.svg
- S2_Icon_Number_20_N.svg
- S2_Icon_ObjectSelectionLasso_20_N.svg
- S2_Icon_Pages_20_N.svg
- S2_Icon_Paragraph_20_N.svg
- S2_Icon_PerspectiveTransform_20_N.svg
- S2_Icon_PolygonLassoSelect_20_N.svg
- S2_Icon_PossibleClippingMask_20_N.svg
- S2_Icon_ProjectsAddInto_20_N.svg
- S2_Icon_Projects_20_N.svg
- S2_Icon_QuickSelect_20_N.svg
- S2_Icon_RectangleSelect_20_N.svg
- S2_Icon_SelectAll_20_N.svg
- S2_Icon_SelectCircularAdd_20_N.svg
- S2_Icon_SelectCircularSubtract_20_N.svg
- S2_Icon_SelectCircular_20_N.svg
- S2_Icon_SelectMultiple_20_N.svg
- S2_Icon_SelectNone_20_N.svg
- S2_Icon_Skew_20_N.svg
- S2_Icon_SmallCaps_20_N.svg
- S2_Icon_Smudge_20_N.svg
- S2_Icon_SubtractFromSelection_20_N.svg
- S2_Icon_SubtractFrom_20_N.svg
- S2_Icon_TextBaselineShift_20_N.svg
- S2_Icon_TextBulleted_20_N.svg
- S2_Icon_TextJustifyLastRight_20_N.svg
- S2_Icon_TextKerning_20_N.svg
- S2_Icon_TextTracking_20_N.svg
- S2_Icon_Transform_20_N.svg
- S2_Icon_Ungroup_20_N.svg
- S2_Icon_Unpublish_20_N.svg
- S2_Icon_Warp_20_N.svg
- S2_Icon_Workspace_20_N.svg
</details>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .changeset/warm-colts-applaud.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@adobe/spectrum-css-workflow-icons": major
+---
+
+chore: take in latest `@a4u/a4u-s2-icon-global-set-open-source-processed`, which is `v5.3.3`.
```
Comment (castastrophe) [inline .changeset/warm-colts-applaud.md]: Should we update this to be more descriptive? Maybe link to the same resource you did the PR description?

**Review by castastrophe — APPROVED** [review]
One note on the changeset description but as far as I can tell, all the required assets updated as expected - components, react, the spritesheet, the svgs, and the manifest. 👍

File: .changeset/warm-colts-applaud.md (hunk 1 above)
Comment (castastrophe) [reply .changeset/warm-colts-applaud.md]: Oh with a Breaking change notice?

Comment (pfulton) [reply .changeset/warm-colts-applaud.md]: Ok, I updated the changeset to include the `breaking change` note and to be more explicit. I also copied over the changelog info in the delta between this newest release and the `2.0.0` release and put it inside of a `<details>` in this PR description and then linked to this PR in the changeset.

File: .changeset/warm-colts-applaud.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+"@adobe/spectrum-css-workflow-icons": major
+---
+
+BREAKING CHANGE: consumes `v3.5.3` (released on Aug 30, 2024) of `@a4u/a4u-s2-icon-global-set-open-source`, bringing this project up-to-date from several previous major version releases which included additions, changes, and removals.
```
Comment (castastrophe) [inline .changeset/warm-colts-applaud.md:5]: Love this! Great detail here but still succinct.

**Review by castastrophe — APPROVED** [review]

**Pull request merged**